contracts/foundry.toml

```diff
@@ -1,8 +1,9 @@
 [profile.default]
-solc = '0.8.16'
+solc = '0.8.25'
 src = "src"
 out = "out"
 libs = ["lib"]
+evm_version = "cancun"
```

Comment:
cancun is live on all the chains we deploy on except scroll, and they have an ETA of early April